src/test/java/com/ncc/aif/ExamplesAndValidationTest.java

```diff
@@ -1475,6 +1476,46 @@ void valid() {
                 testValid("NISTHypothesis.valid: there should be exactly 1 hypothesis");
             }
         }
+
+        @Nested
+        class EntityClusterRequiredHandle {
+            @Test
+            // No handle property on entity cluster in hypothesis
+            void invalidNoHandle() {
+                Resource newEntity = makeEntity(model, getEntityUri(), system);
+                addType(newEntity, SeedlingOntology.Person);
+                makeClusterWithPrototype(model, getClusterUri(), newEntity, system);
+                markJustification(addType(newEntity, SeedlingOntology.Person), justification);
+                makeHypothesis(model, getUri("hypothesis-1"), Collections.singleton(newEntity), system);
+
+                testInvalid("NISTHypothesis.invalid: Each entity cluster in the hypothesis graph must have " +
+                        "exactly one handle");
+            }
+
+            @Test
+            // Two handle properties on entity cluster in hypothesis
+            void invalidMultipleHandles() {
+
+                Resource newEntity = makeEntity(model, getEntityUri(), system);
+                addType(newEntity, SeedlingOntology.Person);
```

Comment:
Type added below. Please remove this line

Reply:
Updated.